polymer.json

```diff
@@ -3,5 +3,6 @@
     "rules": [
       "polymer-2-hybrid"
     ]
-  }
+  },
+  "moduleResolution": "node"
```

Comment:
@awikkerink Interesting... still looking at the rest of the PR. Was kind of aware when I was pulling this store stuff out of the parallel-lms that it was very similar to redux in intent, but decided to keep it mostly unchanged initially.

So generally supportive of this approach.

Out of interest, how does the ES6 integration play nice with the brightspace-integration build? Is that moduleResolution property (presumably a Polymer 3-ish enhancement to the polymer build tools?) all that is required to get transpilation/bundling working with a Polymer 1 app?

Reply:
polymer should bundle the es6 imports with the polymer 3 support. I haven't tested it yet. Just something I'm playing around with